iso_templates/initrd_init_template

```diff
@@ -104,15 +106,45 @@ mount_root() {
     fi
 }
 
-# Finds the installer media and calls mount_root if successful
-find_and_mount_installer() {
+wait_for_keystrokes() {
+    while [ true ] ; do
+        read -t 2 -n 1
+        if [ "$?" = 0 ] ; then
```

Comment:
```suggestion
        if [ "$?" -eq 0 ] ; then
```

Reply:
instead of the loop forever and the read with timeout, why not just the read?
`read -n 1`

Reply:
I guess I can do that. Also will help saves cpu cycles as it would block on input

Reply:
> I guess I can do that. Also will help saves cpu cycles as it would block on input

assuming we keep the wait for the input section

Reply:
Done